.gitignore

```diff
@@ -1,14 +1,157 @@
 # Cargo files
-/target/
 /coverage-target/
-# Firebase caches (?)
-.firebase/
-# Emacs detritus
-*~
-\#*
-*\#
-# Vscode detrius
-.vscode/
+# Zebra state
 .zebra-state/
```

Comment:
This path isn't used any more, but it's still important to ignore it:
```suggestion
# Legacy Zebra state (alpha versions only)
.zebra-state/
```


https://github.com/ZcashFoundation/zebra/blob/d076b999f317b37489d7a6c711d2afdbd03a08c4/zebra-state/src/config.rs#L23-L29